src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -92,8 +92,10 @@ public static void showDownloadMapsWindowAndDownload(final Collection<String> ma
         dia.setLocationRelativeTo(null);
         dia.setMinimumSize(new Dimension(200, 200));
         dia.setVisible(true);
-        dia.requestFocus();
-        dia.toFront();
+        SwingUtilities.invokeLater(() -> {
```

Comment:
Anticipating what I will think if I see this code six months from now :smile:, do you think a short comment would be helpful here?  For example:

```java
// ensure Download Maps window is displayed on top (#1260)
```